Common/Source/InputEvents.cpp

```diff
@@ -2105,6 +2105,13 @@ void InputEvents::eventService(const TCHAR *misc) {
 	return;
   }
 
+  if (_tcscmp(misc, TEXT("AIRSPACELU")) == 0) {
+      LKSound(TEXT("LK_KLICK.WAV"));
+       dlgAirspaceSelect();
+        return;
+  }
+
```

Comment:
why not use CustomKeyHandler ?
like that you avoid to have two different code for same feature ....1